app/views/change_requests/show.slim

```diff
@@ -84,7 +86,8 @@ javascript:
             })
           },
           error: function(response) {
-            bootbox.alert("Cancel Change Request Failed <br><br>" + response.statusText + " - try refreshing the page and trying again");
+            bootbox.hideAll();
```

Comment:
Are there any bootbox alerts that can still show up on this page such that you need the hideAll since you call makeAlert on all of these?